switchboard/models.py

```diff
@@ -34,7 +34,13 @@ def _key(key=''):
     '''
     Returns a Datastore key object, prefixed with the NAMESPACE.
     '''
-    return datastore.Key(os.path.join(NAMESPACE, key))
+    if not isinstance(key, datastore.Key):
+        # Switchboard uses ':' to denote one thing (parent-child) and datastore
+        # uses it for another, so replace ':' in the datastore version of the
+        # key.
+        safe_key = key.replace(':', '|')
```

Comment:
At first I thought there should be some place that reversed this logic to put `|` back to `:` but I suppose we just use the data that is stored, not the key name.  And when querying or doing a lookup, it is done by `Key` so that goes through this too.